src/style.css

```diff
@@ -0,0 +1,81 @@
+@font-face {
+  font-family: "avenir";
+  src: url(fonts/Avenir-Book.ttf) format("woff2"), url(fonts/Avenir-Heavy.ttf) format("woff");
+  font-weight: 300px;
+  font-style: normal;
+}
+
+body {
+  margin-top: 0;
+  padding-top: 20px;
+}
+
+.search-bar {
+  position: relative;
+  display: flex;
+}
+
+.search-bar--big {
+  height: 70px;
+}
+
+.search-bar--small {
+  margin-top: 20px;
+  height: 42px;
+}
+
+.search-bar__input {
+  margin: 0;
+  padding: 0;
+  width: 100vw;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  background: #fff;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+  font-family: "avenir", sans-serif;
+  font-weight: 400;
```

Comment:
400 is a default font-weight

Reply:
[Requirements:]

....
default font-weight must be 300
.....

I'll fix 300px to 300 at @font-face